src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { TodoStatus } from '../../types/TodoStatus';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+interface FooterProps {
+  todos: Todo[];
+  selectedStatus: TodoStatus;
+  onStatusChange: (status: TodoStatus) => void;
+  filteringTodosByActiveStatus: number;
+  TodoStatusRoutes: Record<TodoStatus, string>;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  selectedStatus,
+  onStatusChange,
+  filteringTodosByActiveStatus,
+  TodoStatusRoutes,
+}) => {
+  if (todos.length === 0) {
+    return null;
+  }
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {filteringTodosByActiveStatus} item{filteringTodosByActiveStatus !== 1 ? 's' : ''} left
```

Comment:
`{filteringTodosByActiveStatus !== 1 ? 's' : ''}` - Create a variable for this logic